src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -190,7 +191,7 @@ class AirspeedModule : public ModuleBase<AirspeedModule>, public ModuleParams,
 		(ParamFloat<px4::params::ASPD_SCALE_3>) _param_airspeed_scale_3,
 		(ParamInt<px4::params::ASPD_PRIMARY>) _param_airspeed_primary_index,
 		(ParamInt<px4::params::ASPD_DO_CHECKS>) _param_airspeed_checks_on,
-		(ParamInt<px4::params::ASPD_FALLBACK_GW>) _param_airspeed_fallback_gw,
+		(ParamInt<px4::params::ASPD_FALLBACK>) _param_airspeed_fallback,
```

Comment:
Param translation is required here, otherwise someone not using the default value will have its config silently changed.